install-dev/index.php

```diff
@@ -26,6 +26,10 @@
 
 require_once 'install_version.php';
 
+echo getenv('CI');echo PHP_EOL;
```

Comment:
@matks I think you wanted to modify `index-cli` instead?

Reply:
No, I wanted to see if I can retrieve the value in web, not CLI 😉